### ISSUE TITLE
[forms] "Pay with" Credit card screen transitions

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/klimadao#1424]
Related to #1422

[See design flow here](https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/Projects?type=design&node-id=890-29468&mode=design&t=8dyo3qp0Bv3QcM7Z-4).

After clicking RETIRE NOW from pool listing project page, the form page [loads like this](https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/Projects?type=design&node-id=1076-40839&mode=design&t=8dyo3qp0Bv3QcM7Z-4).
Note that no payment method is selected at this point and we show USDC pricing and info in the Total price component.

When the user selects Credit card as payment we want to add a UI animation to highlight the implications of that selection.
[Design page here](https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/Projects?type=design&node-id=1076-39620&mode=design&t=8dyo3qp0Bv3QcM7Z-4).

**NOTES:**
- the USDC symbol slowly fades out, transition-duration: 800ms
- processing fee section is revealed
- total cost text color switches to green (transition duration: 0ms), then slowly faded to black (transition duration: 800ms)

See professor's comments in the total price component:
![Screenshot 2023-08-14 at 4 35 32 PM](https://github.com/KlimaDAO/klimadao/assets/103535195/cc23db81-f1da-4da1-8039-0d45021d422a)

**Comment by jabby09** [comment]
@0xMakka you may need further info on this one, if yes please discuss with @theprofessora in terms of the transition behaviours he is looking for.

**Comment by 0xMakka** [comment]
Hi @theprofessora,

I've been working on this task, and not entirely sure what the expected behaviour is when the usdc icon disappears. I've added a short video clip of how I have it currently working. I added a transition to the values sliding to the left because without the transition it didn't look quite right when the icon disappears and the value moves to where the icon was (see video). Happy to change this based on your feedback.


https://github.com/KlimaDAO/klimadao/assets/92357475/16c048ae-d99c-4909-9ad4-c60d2b56d4b4
